Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -189,6 +199,28 @@ class BrowserViewController: UIViewController {
             }
             footer.addSubview(footerBackground!)
         }
+        
+        if showTopTabs {
+            if topTabsContainer.subviews.count == 0 {
+                addChildViewController(topTabsViewController)
+                topTabsViewController.view.frame = topTabsContainer.frame
+                topTabsContainer.addSubview(topTabsViewController.view)
+                topTabsViewController.view.snp_makeConstraints { make in
+                    make.edges.equalTo(topTabsContainer)
+                    make.height.equalTo(40)
```

Comment:
Extract this constant into a let variable - maybe in a TopTabsUX structure in `TopTabsViewController`?